src/Sylius/Component/Resource/composer.json

```diff
@@ -26,10 +26,11 @@
         "doctrine/common": "~2.3",
         "symfony/event-dispatcher": "^2.7",
         "symfony/property-access": "^2.7",
-        "winzou/state-machine": "~0.1"
+        "winzou/state-machine": "~0.1",
+        "white-october/pagerfanta-bundle": "^1.0"
```

Comment:
Good catch, probably wrong copy paste, I will check but I am 99% sure we don't need a bundle here.